### ISSUE TITLE
Enhancement: Markdown WYSIWYG tools for Content Editors

### ISSUE THREAD

**Issue by iveptawebadmin** [estruyf/vscode-front-matter#194]
**Is your feature request related to a problem? Please describe.**

I have some editors who are not yet familiar with Markdown syntax and asking them to learn it while trying to get some content into the files is a barrier to entry. 

**Describe the solution you'd like**

It would be great if there was a way to edit the 'content' part of the markdown files with a basic tool set for links, bullets, bold, text formats (headings, paragraph, pre, etc.), code blocks, etc.  Something like what I see at the top of the editing window here in GitLab.

![image](https://user-images.githubusercontent.com/30990811/143499737-e98d718e-e2c4-4b12-bcdc-93276088daf1.png)

**Describe alternatives you've considered**

- Teaching them Markdown and moving on with our lives. The only problem is that the editors in this organization are transient and I would be teaching new people on a regular basis.)
- Using another extension that adds WYSIWYG tools when editing markdown files.

**Comment by estruyf** [comment]
I've added the first support for this in the latest beta version. At the moment, I added:

- bold
- italic
- strikethrough
- code
- codeblock

<img width="327" alt="Screenshot 2021-11-30 at 15 11 48" src="https://user-images.githubusercontent.com/2900833/144062934-90270428-99ad-46c5-b611-68d20007c60d.png">

Many more to come, but this already shows its potential.

Controls can also be disabled with the `frontMatter.content.wysiwyg` setting.

**Comment by zivbk1** [comment]
Excellent! Thank you!

**Comment by estruyf** [comment]
- [ ] Headings
- [ ] Blockquote
- [ ] Ordered list
- [ ] Unordered list

**Comment by estruyf** [comment]
Blockquote + headings added

![Screenshot 2021-12-03 at 13 58 42](https://user-images.githubusercontent.com/2900833/144606521-9f0c6f65-014a-409f-86af-c21244fe8729.png)

![Screenshot 2021-12-03 at 13 59 00](https://user-images.githubusercontent.com/2900833/144606527-a91efdb4-6264-4e20-b1b9-a8594229971d.png)

**Comment by estruyf** [comment]
All list types have been added as well. When you select the text, it will automatically append the selected list markup in front of all the lines.

I had to make a menu, as the title row is limited to a maximum of 10 buttons/icons.

![image](https://user-images.githubusercontent.com/2900833/144651552-289174ca-650d-4361-953a-18e7a2e292d8.png)

![Screenshot 2021-12-03 at 19 02 39](https://user-images.githubusercontent.com/2900833/144651571-a9ef573b-6534-4cc4-a09b-7e6dfe1fbd3b.png)
